index.js

```diff
@@ -180,15 +180,17 @@ module.exports = {
             4,
             {
                 'CallExpression': {
-                    arguments: 'off'
+                    arguments: 1
                 },
+                'flatTernaryExpressions': false,
```

Comment:
I think `'flatTernaryExpressions': false` is how we format. To be exact though, I find the formatting of the example not clear enough and I usually do (presuming all of these are long enough to actually need breaking on multiple lines):

````javascript
var a
    = foo
        ? bar
        : baz
            ? qux
            : boop;
````

Reply:
Here is the rule applied when it doesn't need double line breaks:
![image](https://user-images.githubusercontent.com/9974975/41839211-e766818e-7862-11e8-8578-a25b22863c78.png)

And the same with double breaks:
![image](https://user-images.githubusercontent.com/9974975/41839234-f67e705a-7862-11e8-9912-bf7c05b7ce83.png)

And the rule failed:
![image](https://user-images.githubusercontent.com/9974975/41839248-014a48ce-7863-11e8-8dc7-faf772c29f88.png)

Reply:
Question: do we want to allow single line breaks in ternary expressions? E.g. the line is long enough to break at least once, then do we want to break it into 3 lines:
```javascript
condition
  ? expression1
  : expression2
```
As opposed to
```javascript
condition ? expression1
  : expression2
```